Fails here on Darwin for me: 
```
llvm-config: unknown component name: windowsmanifest
CMake Error at cmake/Modules/FindLLVM.cmake:50 (message):
  Failed to execute llvm-config
  ('/nix/store/9g4chijh9255y65vf3nsgg9258knli58-llvm-5.0.2/bin/llvm-config',
  result code: '1)'
Call Stack (most recent call first):
  cmake/Modules/FindLLVM.cmake:97 (_LLVM_FAIL)
  cmake/Modules/FindLLVM.cmake:122 (llvm_set_libs)
  CMakeLists.txt:34 (find_package)


-- Configuring incomplete, errors occurred!
See also "/private/var/folders/d5/5ybzst6d2wbggcfhkjwl56s80000gn/T/nix-build-ldc-1.24.0.drv-0/ldc-1.24.0-src/build/CMakeFiles/CMakeOutput.log".
builder for '/nix/store/5k9zp5dzbw8s09qlv4a5jlv34s3hz0zh-ldc-1.24.0.drv' failed with exit code 1
error: build of '/nix/store/5k9zp5dzbw8s09qlv4a5jlv34s3hz0zh-ldc-1.24.0.drv' failed
```

--- a/pkgs/development/compilers/ldc/default.nix
+++ b/pkgs/development/compilers/ldc/default.nix
@@ -1,5 +1,4 @@
 import ./generic.nix {
-  version = "1.20.1";
-  ldcSha256 = "1bqsgab22v02pc3c9gcyf15y7aimadv24d68icaw5lpgnvzxy89b";
+  version = "1.24.0";
+  ldcSha256 = "0g5svf55i0kq55q49awmwqj9qi1n907cyrn1vjdjgs8nx6nn35gx";
 }
-

--- a/pkgs/development/compilers/ldc/generic.nix
+++ b/pkgs/development/compilers/ldc/generic.nix
@@ -1,5 +1,5 @@
 { version, ldcSha256 }:
-{ stdenv, fetchurl, cmake, ninja, llvm_5, llvm_8, curl, tzdata
+{ stdenv, fetchurl, cmake, ninja, llvm_11, curl, tzdata
 , libconfig, lit, gdb, unzip, darwin, bash
 , callPackage, makeWrapper, runCommand, targetPackages
 , ldcBootstrap ? callPackage ./bootstrap.nix { }
@@ -56,16 +56,12 @@ stdenv.mkDerivation rec {
   '';
 
   nativeBuildInputs = [
-    cmake ninja makeWrapper unzip ldcBootstrap lit lit.python
+    cmake ldcBootstrap lit lit.python llvm_11 makeWrapper ninja unzip
   ]
   ++ stdenv.lib.optionals stdenv.hostPlatform.isDarwin [
     darwin.apple_sdk.frameworks.Foundation
-    # https://github.com/NixOS/nixpkgs/issues/57120
-    # https://github.com/NixOS/nixpkgs/pull/59197#issuecomment-481972515
-    llvm_5
   ]
   ++ stdenv.lib.optionals (!stdenv.hostPlatform.isDarwin) [
-    llvm_8
     # https://github.com/NixOS/nixpkgs/pull/36378#issuecomment-385034818
     gdb
   ];

